conan/cli/commands/new.py

```diff
@@ -15,7 +15,7 @@ def new(conan_api, parser, *args):
     Create a new recipe (with conanfile.py and other files) from a predefined template
     """
     parser.add_argument("template", help="Template name, built-in predefined one or user one. "
-                        "You can use built-in templates: cmake_lib, cmake_exe, "
+                        "You can use built-in templates: basic, cmake_lib, cmake_exe, "
```

Comment:
The ``basic`` name doesn't match the ``barebones`` name above.
I don't love the ``barebones`` as a ``conan new barebones``, and I think I'd prefer ``conan new basic``, but naming is hard...

Reply:
IMO `basic` is better than `barebones` for sure (what bones? sacrums? and why are they bare? xD)
I would avoid slang terms as much as possible, as they might confuse non-native speakers